src/Ractive/initialise.js

```diff
@@ -36,23 +38,20 @@ export default function initialise ( ractive, userOptions, options ) {
 			}
 		}
 
-		ractive.fragment = new Fragment({
+		ractive.fragment = fragment = new Fragment({
 			owner: ractive,
 			template: ractive.template,
 			cssIds,
 			indexRefs: options.indexRefs || {},
 			keyRefs: options.keyRefs || {}
 		});
+
+		fragment.bind( ractive.viewmodel );
```

Comment:
`.bind(...)` returns `this`, so you can – if you want – do `fragment = new Fragment(...).bind(...)`